dts/msm8226/msm8226-samsung-ms01-r4.dts

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id =  <0x9e00ff01 0x04 0x20000 0x9e08ff01 0x04 0x20000 0xdd08ff01 0x04 0x20000>;
+	
+	ms01 {
+		model = "Samsung Galaxy S3 Neo";
+		compatible = "samsung,ms01", "qcom,msm8226", "lk2nd,device";
```

Comment:
Is there something in the device cmdline that could be used as a match rule here?

Reply:
I'd still appreciate having some match rule for this, could you share the cmdline you get in the lk log maybe?